lib/src/main/scala/spinal/lib/eda/xilinx/VivadoConstraintWriter.scala

```diff
@@ -0,0 +1,171 @@
+package spinal.lib.eda.xilinx
+
+import spinal.core.ClockDomain.FixedFrequency
+import spinal.core.internals._
+import spinal.core.{crossClockMaxDelay, _}
+import spinal.lib.AnalysisUtils
+
+import java.io.{File, PrintWriter, Writer}
+import scala.collection.mutable
+import scala.language.postfixOps
+
+object VivadoConstraintWriter {
+  def apply[T <: Component](
+                             report: SpinalReport[T],
+                             filename: String = null,
+                             trustSpinalTimings: Boolean = false
```

Comment:
remove this since it's currently not implemented?

Reply:
done